### ISSUE TITLE
:bug: Playlists in a Custom_Submenu widget do not play

### ISSUE THREAD

**Issue by L0bsang** [jurialmunkey/skin.arctic.fuse#564]
### Skin section

Widgets

### Current Behavior

I have some smart playlists for oneclick viewing made from my library items. These library items were added via TMDB Helper using custom Trakt lists.

In complex mode I can add shortcuts to these playlists as submenu items and they play fine when clicked.
However in classic mode, there doesn't seem to be any submenus, so I instead I used the experimental "Custom_Submenu" widget to add shortcuts to my playlists. 

The playlists do not play from the custom submenu. 
The action is set to PlayMedia(special://profile/playlists/video/MyPlaylist.xsp) and I just get a "Playback failed" message, with no error message in the logs.

### Expected Behavior

I expect to be able to add shortcuts for my playlists to a custom submenu and click the playlist item to start playing it. I cannot add all my playlists to an ordinary widget because I would like a custom subset and order.

### Steps To Reproduce

1. Add playlist items to a custom submenu widget
2. Choose "Play" as the action instead of "Browse"
3. Try to play the playlists from the submenu

### Screenshots and Additional Info

![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/162974281/72b28233-0d9d-43ee-8e49-051465b5a399)

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
It's okay I can recreate the issue myself, so no need for a log. Just figuring out why it's not working. I think it might be a target issue.

**Comment by L0bsang** [comment]
Thank you for attending to my request! 

I'm not usually one to complain (been lurking for almost 3 years and only recently made an account specifically for this) but family was on my case about their RandomTV not working properly haha.
I see you have made a commit for this bug, but that was an hour after the commit that bumped the version to 0.5.10, so bug still present on my setup (Omega RC1).

Will wait patiently for the 0.5.11 release, and cross my fingers in hope that it successfully fixes this. Thanks again!

**Comment by jurialmunkey** [comment]
Yeah didn't quite make it in to last one but fairly confident it has fixed it since I was able to recreate the issue myself to test and it is working correctly for me now.

**Comment by L0bsang** [comment]
Indeed, fully fixed. Thanks again!
